tests/publiccloud/storage_perf.pm

```diff
@@ -73,6 +74,14 @@ sub run {
     my $disk_size = get_var('PUBLIC_CLOUD_HDD2_SIZE');
     my $disk_type = get_var('PUBLIC_CLOUD_HDD2_TYPE');
     my $url = get_var('PUBLIC_CLOUD_PERF_DB_URI');
+    my $use_nvme = is_azure() && get_var('PUBLIC_CLOUD_INSTANCE_TYPE') =~ 'Standard_L(8|16|32|64)s_v(2|3)';
+
+    if ($use_nvme) {
```

Comment:
I thinks this `record_info` block is not needed, as it's anyways present in line 152 where the target block device is recorded.

Reply:
I wanted to have a label which you can see w/o need to click on anything . I can drop this one and replace line 152 with : 

`record_info( $use_nvme ? `NVMe ON` ?  'NVMe OFF', "Block device under test: $block_device");`  

will it work for you ?